Student Section

```diff
@@ -0,0 +1,12 @@
+- name: Shivam Rai
+  country:India
+  github: ShivamRai2003
+  image: Picture 090.jpg
+  twitter: Shivam Rai
```

Comment:
Just add these lines in the students.yml file in the _data folder.
You don't have to write the rest

Reply:
how to upload a photo there is no option

Reply:
@ShivamRai2003 just add the image to img/students/ directory of the forked repository.